ruby/bps-stan/spec/bps/stan_spec.rb

```diff
@@ -12,20 +12,6 @@ def nats_servers_with_scheme
   nats_servers.map {|s| "nats://#{s}" }
 end
 
-run_spec = \
-  begin
-    # passing a block means that client will be automatically closed once block exits:
-    opts = {
-      servers: nats_servers_with_scheme,
-      dont_randomize_servers: true,
-    }
-    ::STAN::Client.new.connect(TEST_NATS_CLUSTER_ID, TEST_NATS_CLUSTER_ID, nats: opts) {}
```

Comment:
also means that you can get rid of consts and just use local `let(...)`

Reply:
These are used in `helper` block; or you suggest to move it into RSpec.describe?

```ruby
RSpec.describe ... do
  let(:cluster_id) { ... }
  let(:client_id) { ... }
  let(:nats_servers) { ... }

  let(:helper) { ... }
end
```

Not sure if it'll work though.

Reply:
you don't need helper, you can just have an inline `def read_messages` in the `RSpec.describe` block